### PR TITLE
DS 860: Modal docs

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -18,15 +18,15 @@ const Modal: FC<ModalProps> = ({
   imgSrc,
   ...rest
 }) => {
-  const isHeader =
-    header || closeButtonProps?.icon || closeButtonProps?.iconName;
+  const isHeader = header || !!closeButtonProps;
+  const isLargeVariant = size === "lg";
 
-  const variantsStyle = {
+  const styleVariants = {
```
Comment (monikaracka) [inline packages/design-system/src/components/Modal/Modal.tsx:24]: The previous name was more appropriate. These are styles for different variants.

File: packages/design-system/src/components/Modal/Modal.tsx [hunk 2]
```diff
@@ -18,15 +18,15 @@ const Modal: FC<ModalProps> = ({
...
     shadow: "shadow-modal-shadowBase",
     outline: "border-modal-borderWidthBase border-modal-outline",
   };
 
-  const sizeVariantsStyle = {
+  const sizeVariants = {
```
Comment (monikaracka) [inline packages/design-system/src/components/Modal/Modal.tsx:29]: These aren't size variants but styles in terms of size variants.

File: packages/design-system/src/stories/atoms/Modal/Modal.docs.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import { StoryDocsCodeBlock } from "stories/docsHelpers/StoryDocsCodeBlock";
+
+export const ModalDocs = () => (
+  <div>
+    <h1>How to use DS Pagination</h1>
+    <p>
+      To use TimePicker, you have to pass <b>required props</b> to the
+      component:
```
Comment (monikaracka) [inline packages/design-system/src/stories/atoms/Modal/Modal.docs.tsx:8]: Pagination? Timepicker? 😉

Comment (zuzannanap) [reply packages/design-system/src/stories/atoms/Modal/Modal.docs.tsx:8]: ok, double fail :D

**Review by monikaracka — APPROVED** [review]

**Pull request merged**